packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -517,18 +548,22 @@ contract Deploy is Deployer {
 
         // Now that the owners have been added and the threshold increased we can deploy the liveness module (otherwise
         // constructor checks will fail).
-        address module = deployLivenessModule();
+        address livenessModule = deployLivenessModule();
 
         // Unfortunately, a threshold of owners is required to actually enable the module, so we're unable to do that
         // here, and will settle for logging a warning below.
+        // _callViaSafe({ _safe: safe, _target: address(safe), _data: abi.encodeCall(ModuleManager.enableModule,
+        // (module)) });
+        // console.log("LivenessModule enabled on SecurityCouncilSafe");
```

Comment:
Delete these commented out lines of code?

Reply:
TBH I thought they might be helpful for expressing the necessary calls, but I think the WARNING message below is pretty informative.